### PR TITLE
modified models

### PR DESCRIPTION
**Pull request by rutaba1**

### CODE REVIEW

File: lib/core/db_models/old/schema/staff_personal_detail_schema.dart [hunk 1]
```diff
@@ -9,8 +9,8 @@ class StaffPersonalDetailSchema {
   static const String checks = "checks";
   static const String building = "building";
   static const String street = "street";
-  static const String county = "county";
-  static const String city = "city";
-  static const String country = "country";
-  static const String postalCode = "postal_code";
+  static const String houseNumber = "house_number";
```
Comment (SamDuncanAsomas) [inline lib/core/db_models/old/schema/staff_personal_detail_schema.dart:12]: guessing this is a mistake and we shouldn't have an underscore between these two words/

File: lib/core/db_models/old/schema/staff_professional_detail_schema.dart [hunk 2]
```diff
@@ -2,6 +2,6 @@ class StaffProfessionalDetailSchema {
   static const String skills = "skills";
   static const String checks = "checks";
   static const String qualifications = "qualifications";
-  static const String workHistory = "work_history";
+  static const String workHistories = "work_histories";
```
Comment (SamDuncanAsomas) [inline lib/core/db_models/old/schema/staff_professional_detail_schema.dart:5]: guessing this is a mistake and we shouldn't have an underscore between these two words

File: lib/core/db_models/old/staff_personal_detail_model.dart [hunk 3]
```diff
@@ -25,27 +25,24 @@ class StaffPersonalDetailModel{
   
   factory StaffPersonalDetailModel.fromMap(Map<String, dynamic> map){
     if(map != null && map.isNotEmpty){
+      print("MAPS_________$map");
```
Comment (SamDuncanAsomas) [inline lib/core/db_models/old/staff_personal_detail_model.dart]: please make standard or remove

**Review by SamDuncanAsomas — CHANGES_REQUESTED** [review]

File: lib/core/db_models/old/schema/staff_personal_detail_schema.dart (hunk 1 above)
Comment (rutaba1) [reply lib/core/db_models/old/schema/staff_personal_detail_schema.dart:12]: its a key name which is coming from api, the variable name does not have any underscore

File: lib/core/db_models/old/schema/staff_professional_detail_schema.dart (hunk 2 above)
Comment (rutaba1) [reply lib/core/db_models/old/schema/staff_professional_detail_schema.dart:5]: its a key name which is coming from api, the variable name does not have any underscore

**Pull request merged**